### PR TITLE
Download project UI improvement (resolves #23)

### LINKED ISSUE #23: Style the "Download Project" page
**Is your improvement related to a problem? Please describe.**
The "Download Project" page (`/download_project/<project.id>`) is bland and looks empty:
image

**Describe the solution or change you'd like**
The page could be improved to look better, while not necessarily having more elements for the moment. Any reasonable styling would be great, I don't have anything too specific in mind.

If you want to do more, some statistics about the project could be displayed next to the form. For example, a reminder of the project name and project author, the number of molecules in the project, possibly the total number of calculations in the project (calculated recursively for each molecule, ensemble and structure). This is not mandatory, but could add some more content on the page.

### PR DESCRIPTION
**Pull request by Omkar0803**

### CODE REVIEW

File: frontend/templates/frontend/download_project.html [hunk 1]
```diff
@@ -19,9 +76,12 @@
         refresh_availabilities();
     });
 </script>
-{% endblock %} {% block content %}
+
 <form action="/download_project/" method="post" id="calcform" enctype="multipart/form-data">
+    <div class="formContainer">
     {% csrf_token %}
+    <h1 id="downloadHeading">Download Project</h1>
+    <h3 id="project_heading">Project Heading</h3>
```
Comment (RaphaelRobidas) [inline frontend/templates/frontend/download_project.html:83]: You should add the name of the project here instead of "Project Heading"

Comment (Omkar0803) [reply frontend/templates/frontend/download_project.html:83]: Ohh, sorry I forgot to add it.

Comment (RaphaelRobidas) [reply frontend/templates/frontend/download_project.html:83]: Could you quickly fix, commit and push it? It will update the PR automatically.

Comment (Omkar0803) [reply frontend/templates/frontend/download_project.html]: I have pushed the changes.

**Pull request merged**